src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
-}
+  let newState = { ...state };
+  const resultArray = [];
```

Comment:
```suggestion
  const result= [];
```

Don't use variable type for naming